application-templates/django-ninja/backend/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pydantic==2.9.2
+django-ninja
```

Comment:
Would it make sense to have cloudharness and cloudharness_django as dependencies here? Since they are conceptually required to run the application (f.e. they are used in settings.py) @ddelpiano

Reply:
I would expect so. I understand we might need CH repo locally if I am in need of regen the template files or the codefresh pipeline, but if I am purely working on the backend I might not need CH and such dependency should be part of the requirements.txt so that all I need to run the backend is already covered by that. Thanks!

Reply:
I agree it's better to be explicit about all the dependencies that are imported, even if they are inherited by the base image (like this case)

Reply:
I don't know how I'd go about adding that to the `requirements.txt` file. The cloud harness modules aren't on pip, so I can't just have `cloudharness-django` in there, and the path to them is not well defined; they're at `/libraries/cloudharness-django` in the base image, but that isn't necessarily going to always be true, and definitely won't be true for running it outside the image.